src/components/Filter/Filter.js

```diff
@@ -0,0 +1,314 @@
+import React, { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark, faMinus } from '@fortawesome/free-solid-svg-icons';
+import RangeSlider from '../RangeSlider/RangeSlider';
+import {
+  StyleFilter,
+  H2,
+  GraySpan,
+  GraySmallSpan,
+  TitleSpan,
+  Header,
+  HeaderDiv,
+  CloseBtn,
+  BoldSpan,
+  Body,
+  BodyDiv,
+  BodyInner,
+  BodyContainer,
+  BodyWrapper,
+  BodyWrapperDiv,
+  FeeDiv,
+  FeeInput,
+  BodyTypeDiv,
+  BodyWrapperInner,
+  BodyTypeFilter,
+  CheckboxInput,
+  BodyTypeFilterInner,
+  Footer,
+  BlackBtn,
+  MoreSpan,
+  MoreDiv,
+  AllCancelDiv,
+  AllCancelSpan,
+  FooterDiv,
+  RangeBox,
+  RangeBoxInner,
+  Slider,
+  RangeDiv,
+} from './StyleFilter';
+
+const Filter = ({ isOpenModal, handleModal }) => {
+  const [isAdd, setIsAdd] = useState(false);
+  const [studioDataList, setStudioDataList] = useState([]);
+  const [payRange, setPayRange] = useState([10000, 40000]);
+  const [checkedList, setCheckedList] = useState([]);
+
+  const handleRange = (event, index) => {
+    const { value } = event.target;
+    const newArray = [...payRange];
+    newArray[index] = value;
+    setPayRange(newArray);
+  };
+
+  const handleChecked = e => {
+    const { checked, name } = e.target;
+    if (checked) {
+      setCheckedList(prev => [...prev, name]);
+    } else {
+      setCheckedList(prev => prev.filter(item => item !== name));
+    }
+  };
+  const handleCheckCancel = () => {
+    setCheckedList([]);
+  };
+  useEffect(() => {
+    fetch('data/filterData.json')
+      .then(response => response.json())
+      .then(result => {
+        setStudioDataList(result.data);
+      });
+  }, []);
+
+  return (
+    <StyleFilter>
```

Comment:
```suggestion
    <S.StyleFilter>
```
위의 리뷰를 토대로 하면 이렇게 적용할 수 있겠습니다!